eng/pipelines/aggregate-reports.yml

```diff
@@ -7,61 +7,22 @@ jobs:
     timeoutInMinutes: 180
     variables:
       - template: templates/variables/globals.yml
+      - name: ProjectList
+        value: com.azure:azure-data-appconfiguration,com.azure:azure-core,com.azure:azure-core-amqp,com.azure:azure-core-http-jdk-httpclient,com.azure:azure-core-http-netty,com.azure:azure-core-http-okhttp,com.azure:azure-core-test,com.azure:azure-core-tracing-opentelemetry,com.azure:azure-cosmos,com.azure:azure-messaging-eventhubs,com.azure:azure-messaging-eventhubs-checkpointstore-blob,com.azure:azure-identity,com.azure:azure-security-keyvault-certificates,com.azure:azure-security-keyvault-keys,com.azure:azure-security-keyvault-secrets,com.azure:azure-storage-blob,com.azure:azure-storage-blob-cryptography,com.azure:azure-storage-blob-nio,com.azure:azure-storage-common,com.azure:azure-storage-file-datalake,com.azure:azure-storage-file-share,com.azure:azure-storage-queue,com.azure:azure-sdk-template,com.azure:azure-ai-textanalytics
```

Comment:
Does this get generated? This concerns me a bit as this seems like another one of those scenarios where we continuously forget to update this as we move libraries around or on-board new libraries.

Reply:
At the moment this is just a dead body in the aggregate-reports.yml file. I'll be deleting that line in a subsequent PR but I want to avoid changes if I can so I don't eat up the agent pool again. Getting the aggregate reporting pipeline working correctly is going to be a phase 3 I think :)